batch/test/test_copy_files.py

```diff
@@ -0,0 +1,547 @@
+import os
+import unittest
```

Comment:
do we have a test for downloading files when there exists a size zero file named `foo/` and also some files named `foo/a` and `foo/b`? Since Hail tables and matrix tables have this structure, I think it's important we test that behavior.

Reply:
It's impossible to create a size zero file named `foo/` just with gsutil alone. Let me see if I can do it with the python batch client.